platform/android/src/style/sources/source.cpp

```diff
@@ -68,6 +107,14 @@ namespace android {
         return std::move(ownedSource);
     }
 
+    jni::jobject* Source::getJavaPeer(jni::JNIEnv& env) {
```

Comment:
Is this called anywhere?

Reply:
That's a 🐛 . It should've been called in `createJavaSourcePeer()`